monitoring/file_read_sample.h

```diff
@@ -0,0 +1,25 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//  This source code is also licensed under the GPLv2 license found in the
+//  COPYING file in the root directory of this source tree.
+//
+#pragma once
+#include "db/version_edit.h"
+#include "util/random.h"
+
+namespace rocksdb {
+static const uint32_t kFileReadSampleRate = 1024;
```

Comment:
I have no idea. We can make it configurable once there is such a case. So far I would keep it hard-coded to avoid yet another option.